src/validator/MaintainValidator.cpp

```diff
@@ -303,61 +302,46 @@ Status DropEdgeValidator::toPlan() {
 }
 
 Status CreateTagIndexValidator::validateImpl() {
-    auto sentence = static_cast<CreateTagIndexSentence *>(sentence_);
-    tagName_ = *sentence->tagName();
-    indexName_ = *sentence->indexName();
-    fields_ = sentence->columns();
+    auto sentence = static_cast<CreateTagIndexSentence*>(sentence_);
+    name_ = *sentence->tagName();
+    index_ = *sentence->indexName();
+    fields_ = sentence->fields();
     ifNotExist_ = sentence->isIfNotExist();
-
-    auto status = Status::OK();
-    do {
-        auto tagStatus = qctx_->schemaMng()->toTagID(space_.id, tagName_);
-        NG_RETURN_IF_ERROR(tagStatus);
-
-        auto schema_ = qctx_->schemaMng()->getTagSchema(space_.id, tagStatus.value());
-        if (schema_ == nullptr) {
-            return Status::SemanticError("No schema found for '%s'", tagName_.c_str());
-        }
-
-        status = IndexUtil::validateColumns(fields_);
-        if (!status.ok()) {
-            VLOG(1) << status;
-            break;
-        }
-    } while (false);
```

Comment:
Why remove these codes?

Reply:
> Why remove these codes?

No need to check in graph layer, this piece of code is already implemented in the meta layer.
And the meta cache's interval wait is avoided.